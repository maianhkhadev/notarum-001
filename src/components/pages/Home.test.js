import React from 'react';
import renderer from 'react-test-renderer';
import Page from './Home';

it('renders correctly', () => {
  const tree = renderer.create(<Page />).toJSON();
  expect(tree).toMatchSnapshot();
});
