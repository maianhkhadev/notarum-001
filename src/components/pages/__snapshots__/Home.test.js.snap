// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="layout layout-default"
>
  <header
    className="layout-header"
  >
    <div
      className="container"
    >
      <nav
        className="navbar"
      >
        <a
          className="navbar-brand"
          href="index.html"
        >
          <span>
            N
          </span>
          otarum
        </a>
        <form
          className="form-inline"
        >
          <button
            className="btn btn-outline-dark"
          >
            Free trial
          </button>
        </form>
      </nav>
    </div>
  </header>
  <main
    className="layout-main"
  >
    <div
      className="page page-home"
    >
      <div
        className="page-header"
      />
      <div
        className="page-content"
      >
        <section
          className="section-hero"
        >
          <div
            className="container"
          >
            <div
              className="row"
            >
              <div
                className="col-xl-6 align-self-center"
              >
                <h1
                  className="section-title"
                >
                  Automating corporate due diligence
                </h1>
                <p
                  className="text"
                >
                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec pellentesque efficitur turpis, vitae dictum dolor tristique in. Mauris tristique id urna at cursus.
                </p>
                <button
                  className="btn btn-primary"
                >
                  Learn More
                </button>
              </div>
              <div
                className="col-xl-6"
              >
                <img
                  alt=""
                  className="w-100"
                  src="process.png"
                />
              </div>
            </div>
          </div>
        </section>
        <section
          className="section-features"
        >
          <div
            className="container"
          >
            <div
              className="row"
            >
              <div
                className="col-xl-4"
              >
                <div
                  className="feature"
                >
                  <img
                    alt=""
                    className="feature-icon"
                    src="login.png"
                  />
                  <div
                    className="feature-content"
                  >
                    <h4
                      className="feature-title"
                    >
                      Faster Processing
                    </h4>
                    <p
                      className="feature-text"
                    >
                      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
                    </p>
                  </div>
                </div>
                <div
                  className="feature"
                >
                  <img
                    alt=""
                    className="feature-icon"
                    src="login.png"
                  />
                  <div
                    className="feature-content"
                  >
                    <h4
                      className="feature-title"
                    >
                      Primary Source Verification
                    </h4>
                    <p
                      className="feature-text"
                    >
                      Lorem ipsum dolor sit amet.
                    </p>
                  </div>
                </div>
              </div>
              <div
                className="col-xl-4"
              >
                <div
                  className="feature"
                >
                  <img
                    alt=""
                    className="feature-icon"
                    src="login.png"
                  />
                  <div
                    className="feature-content"
                  >
                    <h4
                      className="feature-title"
                    >
                      Smart Screening
                    </h4>
                    <p
                      className="feature-text"
                    >
                      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
                    </p>
                  </div>
                </div>
                <div
                  className="feature"
                >
                  <img
                    alt=""
                    className="feature-icon"
                    src="login.png"
                  />
                  <div
                    className="feature-content"
                  >
                    <h4
                      className="feature-title"
                    >
                      Collaborative Workflow
                    </h4>
                    <p
                      className="feature-text"
                    >
                      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
                    </p>
                  </div>
                </div>
              </div>
              <div
                className="col-xl-4 align-self-center"
              >
                <h3
                  className="section-title"
                >
                  Lorem ipsum dolor sit amet consectetur
                </h3>
                <p
                  className="text"
                >
                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec pellentesque efficitur turpis, vitae dictum dolor tristique in. Mauris tristique id urna at cursus.
                </p>
              </div>
            </div>
          </div>
        </section>
        <section
          className="section-services"
        >
          <div
            className="container"
          >
            <div
              className="section-header"
            >
              <div
                className="row"
              >
                <div
                  className="col-xl-8 mx-auto text-center"
                >
                  <h2
                    className="section-title"
                  >
                    Lorem ipsum dolor sit amet consectetur
                  </h2>
                  <p
                    className="text"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec pellentesque efficitur turpis, vitae dictum dolor tristique in. Mauris tristique id urna at cursus.
                  </p>
                </div>
              </div>
            </div>
            <div
              className="section-content"
            >
              <div
                className="row"
              >
                <div
                  className="col-xl-6"
                >
                  <img
                    alt=""
                    className="w-100"
                    src="macbook.png"
                  />
                </div>
                <div
                  className="col-xl-6"
                >
                  <div
                    className="service"
                  >
                    <figure
                      className=""
                    >
                      <img
                        alt=""
                        className=""
                        src="test.png"
                      />
                    </figure>
                    <div
                      className="service-content"
                    >
                      <h4
                        className="service-name"
                      >
                        Efficient
                      </h4>
                      <p
                        className="service-text"
                      >
                        Our customisable workflow means faster onboarding and monitoring, and easy collaboration between teams
                      </p>
                    </div>
                  </div>
                  <div
                    className="service"
                  >
                    <figure
                      className=""
                    >
                      <img
                        alt=""
                        className=""
                        src="test.png"
                      />
                    </figure>
                    <div
                      className="service-content"
                    >
                      <h4
                        className="service-name"
                      >
                        Effective
                      </h4>
                      <p
                        className="service-text"
                      >
                        Rely on primary source data for continuous monitoring so you know exactly who you are doing business with
                      </p>
                    </div>
                  </div>
                  <div
                    className="service"
                  >
                    <figure
                      className=""
                    >
                      <img
                        alt=""
                        className=""
                        src="test.png"
                      />
                    </figure>
                    <div
                      className="service-content"
                    >
                      <h4
                        className="service-name"
                      >
                        Compliant
                      </h4>
                      <p
                        className="service-text"
                      >
                        Smart screening and reports ensure your organisation adopts a risk-based approach to customer due diligence
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </section>
        <section
          className="section-mission"
        >
          <div
            className="container"
          >
            <div
              className="row"
            >
              <div
                className="col-xl-8 mx-auto text-center"
              >
                <h4
                  className="section-title"
                >
                  Our AI platform and workflow tools automate ~90% of the current process reducing cost and risk, a rare win:win
                </h4>
                <p
                  className="text"
                >
                  We aim to service a global web of millions of interconnected companies, whose due diligence needs are entirely automated, suspect characters are flagged and removed, and companies’ regulatory risk eliminated.
                </p>
              </div>
            </div>
          </div>
        </section>
        <section
          className="section-tutorial"
        >
          <h2
            className="section-title"
          >
            Easy to get started, Four Step Process
          </h2>
          <div
            className="container"
          >
            <div
              className="row"
            >
              <div
                className="col-xl-3 col-md-6"
              >
                <div
                  className="step"
                >
                  <img
                    alt=""
                    className="icon"
                    src="login.png"
                  />
                  <h4
                    className="title"
                  >
                    <strong>
                      01.
                    </strong>
                     Access Dashboard
                  </h4>
                  <p
                    className="text"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec et aliquet neque, id pretium risus.
                  </p>
                </div>
              </div>
              <div
                className="col-xl-3 col-md-6"
              >
                <div
                  className="step"
                >
                  <img
                    alt=""
                    className="icon"
                    src="login.png"
                  />
                  <h4
                    className="title"
                  >
                    <strong>
                      02.
                    </strong>
                     Lorem ipsum dolor
                  </h4>
                  <p
                    className="text"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec et aliquet neque, id pretium risus.
                  </p>
                </div>
              </div>
              <div
                className="col-xl-3 col-md-6"
              >
                <div
                  className="step"
                >
                  <img
                    alt=""
                    className="icon"
                    src="login.png"
                  />
                  <h4
                    className="title"
                  >
                    <strong>
                      03.
                    </strong>
                     Lorem ipsum dolor
                  </h4>
                  <p
                    className="text"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec et aliquet neque, id pretium risus.
                  </p>
                </div>
              </div>
              <div
                className="col-xl-3 col-md-6"
              >
                <div
                  className="step"
                >
                  <img
                    alt=""
                    className="icon"
                    src="login.png"
                  />
                  <h4
                    className="title"
                  >
                    <strong>
                      04.
                    </strong>
                     Lorem ipsum dolor
                  </h4>
                  <p
                    className="text"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec et aliquet neque, id pretium risus.
                  </p>
                </div>
              </div>
            </div>
          </div>
        </section>
        <section
          className="section-contact"
        >
          <div
            className="container"
          >
            <div
              className="row"
            >
              <div
                className="col-xl-6"
              >
                <img
                  alt=""
                  className="thumbnail"
                  src="handshake.jpg"
                />
              </div>
              <form
                className="col-xl-6"
              >
                <h3
                  className="section-title"
                >
                  We'd love to hear from you.
                </h3>
                <p
                  className="text"
                >
                  Let us know if you are interested to participate in our beta program. We will then reach out to understand more about your business and requirements.
                </p>
                <div
                  className="form-group"
                >
                  <label>
                    Name
                  </label>
                  <input
                    className="form-control"
                    placeholder="Ex: Mai Anh Kha"
                    type="text"
                  />
                </div>
                <div
                  className="form-group"
                >
                  <label
                    htmlFor="exampleInputEmail1"
                  >
                    Email address
                  </label>
                  <input
                    aria-describedby="emailHelp"
                    className="form-control"
                    id="exampleInputEmail1"
                    placeholder="Enter email"
                    type="email"
                  />
                </div>
                <div
                  className="form-group"
                >
                  <label>
                    Subject
                  </label>
                  <input
                    className="form-control"
                    placeholder="Ex: Contact"
                    type="text"
                  />
                </div>
                <div
                  className="form-group"
                >
                  <label>
                    Message
                  </label>
                  <textarea
                    className="form-control"
                    rows="3"
                  />
                </div>
                <button
                  className="btn btn-primary btn-block"
                  type="submit"
                >
                  Submit
                </button>
              </form>
            </div>
          </div>
        </section>
      </div>
      <div
        className="page-footer"
      />
    </div>
  </main>
  <footer
    className="layout-footer"
  >
    <div
      className="container"
    >
      <section
        className="section-01"
      >
        <div
          className="row"
        >
          <div
            className="col-xl-3"
          >
            <h5
              className="footer-title"
            >
              Products
            </h5>
            <ul
              className="footer-nav"
            >
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  Lorem Ipsum
                </a>
              </li>
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  Lorem Ipsum
                </a>
              </li>
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  Lorem Ipsum
                </a>
              </li>
            </ul>
          </div>
          <div
            className="col-xl-3"
          >
            <h5
              className="footer-title"
            >
              Resources
            </h5>
            <ul
              className="footer-nav"
            >
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  Docs
                </a>
              </li>
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  Guides
                </a>
              </li>
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  Release Notes
                </a>
              </li>
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  Blogs
                </a>
              </li>
            </ul>
          </div>
          <div
            className="col-xl-3"
          >
            <h5
              className="footer-title"
            >
              Company
            </h5>
            <ul
              className="footer-nav"
            >
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  About Us
                </a>
              </li>
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  Contact Us
                </a>
              </li>
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  Career
                </a>
              </li>
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  Customers
                </a>
              </li>
            </ul>
          </div>
          <div
            className="col-xl-3"
          >
            <h5
              className="footer-title"
            >
              Quicklinks
            </h5>
            <ul
              className="footer-nav"
            >
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  Support
                </a>
              </li>
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  Terms
                </a>
              </li>
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  Privacy Policy
                </a>
              </li>
            </ul>
          </div>
        </div>
      </section>
      <section
        className="section-02"
      >
        <div
          className="row"
        >
          <div
            className="col-xl-3"
          >
            <h5
              className="footer-title"
            >
              Get Socials
            </h5>
            <ul
              className="footer-nav"
            >
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  Facebook
                </a>
              </li>
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  Twitter
                </a>
              </li>
              <li
                className="footer-item"
              >
                <a
                  className="footer-link"
                  href="index.html"
                >
                  Linkedin
                </a>
              </li>
            </ul>
          </div>
          <div
            className="col-xl-3"
          >
            <h5
              className="footer-title"
            >
              Singapore
            </h5>
            <div
              className="address"
            >
              32 Carpenter Street 059911 Singapore
            </div>
          </div>
          <div
            className="col-xl-3"
          >
            <h5
              className="footer-title"
            >
              Vietnam
            </h5>
            <div
              className="address"
            >
              32 Carpenter Street 059911 Singapore
            </div>
          </div>
          <div
            className="col-xl-3"
          />
        </div>
      </section>
    </div>
    <div
      className="copyright"
    >
      <span>
        Copyright 2019 Notarum
      </span>
      <span>
        All Rights Reserved
      </span>
    </div>
  </footer>
</div>
`;
